tools/find-inactive-tsc.mjs

```diff
@@ -0,0 +1,260 @@
+    // If we've been processing TSC emeriti and we reach the end of
+    // the list, print out the remaining entries to be moved because they come
+    // alphabetically after the last item.
+    if (inTscEmeritusSection && line === '' &&
+        fileContents.endsWith('>\n')) {
+      while (textToMove.length) {
+        fileContents += textToMove.pop();
+      }
+    }
+
+    // If we've found the TSC heading already, stop processing at the
+    // next heading.
+    if (line.startsWith('#')) {
+      inTscSection = false;
+      inTscEmeritusSection = false;
+    }
+
+    const isTsc = inTscSection && line.length;
+    const isTscEmeritus = inTscEmeritusSection && line.length;
+
+    if (line === '### TSC (Technical Steering Committee)') {
+      inTscSection = true;
+    }
+    if (line === '### TSC emeriti') {
+      inTscEmeritusSection = true;
+    }
+
+    if (isTsc) {
+      if (line.startsWith('* ')) {
+        memberFirstLine = line;
+        const match = line.match(/^\* \[([^\]]+)/);
+        if (match && peopleToMove.includes(match[1])) {
+          moveToInactive = true;
+        }
+      } else if (line.startsWith('  **')) {
+        if (moveToInactive) {
+          textToMove.push(`${memberFirstLine}\n${line}\n`);
+          moveToInactive = false;
+        } else {
+          fileContents += `${memberFirstLine}\n${line}\n`;
+        }
+      } else {
+        fileContents += `${line}\n`;
+      }
+    }
+
+    if (isTscEmeritus) {
+      if (line.startsWith('* ')) {
+        memberFirstLine = line;
+      } else if (line.startsWith('  **')) {
+        const currentLine = `${memberFirstLine}\n${line}\n`;
+        // If textToMove is empty, this still works because when undefined is
+        // used in a comparison with <, the result is always false.
+        while (textToMove[0] < currentLine) {
+          fileContents += textToMove.shift();
+        }
+        fileContents += currentLine;
+      } else {
+        fileContents += `${line}\n`;
+      }
+    }
+
+    if (!isTsc && !isTscEmeritus) {
+      fileContents += `${line}\n`;
+    }
+  }
+
+  return fileContents;
+}
+
+// Get current TSC members, then get TSC members at start of period. Only check
+// TSC members who are on both lists. This way, we don't flag someone who has
+// only been on the TSC for a week and therefore hasn't attended any meetings.
+const tscMembersAtEnd = await getTscFromReadme();
+
+await runGitCommand(`git checkout 'HEAD@{${SINCE}}' -- README.md`);
+const tscMembersAtStart = await getTscFromReadme();
+await runGitCommand('git reset HEAD README.md');
+await runGitCommand('git checkout -- README.md');
+
+const tscMembers = tscMembersAtEnd.filter(
+  (memberAtEnd) => tscMembersAtStart.includes(memberAtEnd)
+);
+
+// Get all meetings since SINCE.
+// Assumes that the TSC repo is cloned in the .tmp dir.
+const meetings = await runGitCommand(
+  `git whatchanged --since '${SINCE}' --oneline --name-only --pretty=format: meetings`,
```

Comment:
`--onleline` is ignored when used with the `--pretty` option.
```suggestion
  `git whatchanged --since '${SINCE}' --name-only --pretty=format: meetings`,
```